src/etc/inc/services.inc

```diff
@@ -2214,7 +2214,7 @@ function dyndnsCheckIP($int) {
 		$ip_result_page = curl_exec($ip_ch);
 		curl_close($ip_ch);
 		$ip_result_decoded = urldecode($ip_result_page);
-		preg_match('=Current IP Address: (.*)</body>=siU', $ip_result_decoded, $matches);
```

Comment:
Note to self: refactor this to split out the regexes per-provider

Should I store a [mapping](https://www.php.net/manual/en/language.types.array.php) of known per-provider regexes for the oddballs as a constant booger in here, or have that passed in via fragment parameter or directive?